partcad/src/partcad/ai_ollama.py

```
@@ -10,19 +10,20 @@
 import base64
 import importlib
 import httpx
-from path import Path
+from pathlib import Path
 import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)